install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,209 @@
+[[install-config-configuring-nsx-t-sdn]]
+[%hardbreaks]
+= Configuring NSX-T SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[nsx-t-sdn-and-openshift]]
+== NSX-T SDN and {product-title}
+
+VMware NSX-T provides advanced software-defined networking (SDN), security and visibility
+to container environments that simplifies IT operations and extends {product-title}’s native
+networking capabilities.
+
+NSX-T supports Virtual Machine, Bare-metal, and container workloads across multiple clusters. This allows
+organisations to have complete visibility using a single SDN across the entire environment.
+
+
+See xref:../architecture/networking/network_plugins.adoc#nsx-sdn[Networking]
+for more information on how NSX-T is integrated with {product-title}.
+
+[[nsx-t-sdn-operations-workflow]]
+== Example Topology
+
+There is a *T0 router* on the top that connects physical with virtual world. We also have *T1 router* acting as a default gateway for the OpenShift VMs.
+
+Those VMs have *two vNICs* each. One vNIC is connected to Management Logical Switch for accessing the VMs. The second vNIC is connected to a dump Logical Switch and is used by nsx-node-agent to uplink the POD networking. For further details, we strongly suggest reading the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.3/nsxt_23_ncp_OpenShift.pdf[NSX Container Plug-in for OpenShift].
+
+The LoadBalancer used for configuring OpenShift Routes plus all project’s *T1 routers* and *Logical Switches* are created automatically later when we install OpenShift. In this topology we use the default OpenShift HAProxy Router for all infrastructure components such as Grafana, Prometheus, Console, Service Catalog and others.
+
+This means that the DNS records for the infra components need to point the infrastructure node IP addresses since the HAProxy uses the host network namespace.
+This works well for infra routes but in order to avoid exposing the infra nodes management IPs to the outside world we will be deploying application specific routes to the NSX-T LoadBalancer.
+
+The topology here assumes 3 x OpenShift master virtual machines and 4 x OpenShift worker virtual machines(two for infrastructure and two for compute).
```

Comment:
@RobbieJVMW Can the user use 4.x and 3.x together? Or are you connecting the two?

Reply:
This is referring to the number of virtual machines not the Openshift version ( if that's the question).  The target for this update is 3.11